scss/numerictextbox/_layout.scss

```diff
@@ -14,4 +14,8 @@
         }
     }
 
+    .k-numeric-wrap .k-i-warning {
+        position: absolute;
+        right: $spacer-x * 2;
```

Comment:
That's definitely not going to look good in bootstrap theme. We'll fix it at a later stage.